src/expression.d

```diff
@@ -15915,6 +15915,38 @@ extern (C++) final class FileInitExp : DefaultInitExp
 
 /***********************************************************
  */
+extern (C++) final class FileFullPathInitExp : DefaultInitExp
+{
+    extern (D) this(Loc loc)
+    {
+        super(loc, TOKfilefullpath, __traits(classInstanceSize, FileFullPathInitExp));
```

Comment:
This class is identical to `FileInitExp` except for `TOKfilefullpath` instead of `TOKfile`. Such copy/pasta of code should not be.